ui/src/pages/Cluster/Detail/Tenant/index.tsx

```diff
@@ -1,11 +1,10 @@
 import EventsTable from '@/components/EventsTable';
 import MonitorComp from '@/components/MonitorComp';
 import TenantsList from '@/pages/Tenant/TenantsList';
-import { useParams } from '@umijs/max';
 import { getClusterDetailReq } from '@/services';
 import { getAllTenants } from '@/services/tenant';
 import { PageContainer } from '@ant-design/pro-components';
-import { useNavigate } from '@umijs/max';
+import { useNavigate,useParams } from '@umijs/max';
```

Comment:
Why there is no space between imported items in this file while there are in some other files. Maybe the style should be unified by lint tool.